M6-Backend/FacilityInterfaces/interfaceNewestFacility.js

```diff
@@ -0,0 +1,22 @@
+
+function interfaceNewestFacility(info){
+    const  testQuery = [];
+    var finalResult={};
+
+    if(info.length == 0){
+        finalResult = {result:"", length:info.length};
+    }
+    else{
+        for(i=0;i<info.length;i++){
+            testQuery[i] = info[i];
+        }
+
+        finalResult = {result:testQuery, length:info.length};
+    }
+
+    return finalResult;
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/CPEN321HELP/CPEN321/pullRequest?prid=9852520)
```suggestion
  return finalResult;
```